config/nav.yml

```diff
@@ -207,7 +206,7 @@ nav:
       - Experimental Features: eventing/experimental-features.md
       - Code samples:
         - Overview: eventing/samples/README.md
-        - Container source: eventing/sources/container-source/README.md
+        - Container source: eventing/samples/container-source/README.md
```

Comment:
I'm not sure why we'd keep this in two places instead of merging it with the other container source docs, but maybe we should open an additional issue to follow that up later

Reply:
Yeah, I think we need to decide what should be in the samples and what should be documentation. Didn't really want to decide that here so I just moved everything that was an example into the samples folder.